about.toml

```diff
@@ -0,0 +1,20 @@
+accepted = [
+    "Apache-2.0",
+    "MIT",
+    "Apache-2.0 WITH LLVM-exception",
+    "BSD-2-Clause",
+    "CC0-1.0",
+    "BSD-3-Clause",
+    "MPL-2.0",
+    "ISC",
+    "OpenSSL",
+    "Unicode-DFS-2016",
+    "Zlib",
```

Comment:
So GPL licences allow that many more licences? (comparing to pallets https://github.com/integritee-network/pallets/blob/917dd4f241ea41a50231c1effb060ab38745ad7d/about.hbs)